Microsoft.Toolkit.HighPerformance/Extensions/ArrayExtensions.1D.cs

```diff
@@ -203,7 +210,7 @@ public static bool IsCovariant<T>(this T[] array)
         /// <summary>
         /// Throws an <see cref="OverflowException"/> when the "column" parameter is invalid.
         /// </summary>
-        public static void ThrowOverflowException()
+        private static void ThrowOverflowException()
```

Comment:
Was this `public` in 6.1? This is just an internal helper, eh?

Reply:
Thankfully not 🤣
This is just an internal helper that I added in #3353, but that accidentally left out as public there. I think it was because this was initially in an internal throw helper class (so the method was public), then I moved it here and forgot to change the accessibility modifier. Really glad I spotted it in time for 7.0 😅